Source/VCGeneration/VCGen.cs

```diff
@@ -360,11 +360,6 @@ record ImplementationTransformationData {
       callback.OnProgress?.Invoke("VCgen", 0, 0, 0.0);
 
       Stopwatch watch = new Stopwatch();
```

Comment:
Is this `Stopwatch` still used anywhere after removing the code below?

Reply:
Nope, it's a rebase gone wrong.  Thanks!